module/support/logging/DataLogging/src/DataLogging.cpp

```diff
@@ -158,6 +159,20 @@ namespace module::support::logging {
                     handles = std::move(new_handles);
                 }
             });
+
+        on<Every<10, Per<std::chrono::seconds>>, Sync<DataLog>>().then([this]() {
+            unsigned int size = 0;
+            for (auto& f : std::filesystem::recursive_directory_iterator(config.output.directory)) {
```

Comment:
I am pretty sure this is a far more expensive call than we want to be running 10 times per second. I reckon once every 5 seconds would probably do lol.